multiqc/utils/config_defaults.yaml

```diff
@@ -232,6 +232,9 @@ fn_clean_exts:
   - ".mpileup"
   - ".primer_trim"
   - ".mapped"
+  - ".vep"
+  - "_vep"
+  - ".txt_summary"
```

Comment:
We already have `".txt"` further up, so I don't think that we need this one - please correct me if I'm wrong.

```suggestion
```

Reply:
I wasn't sure if `".txt"` would clean up `".txt_summary"` so I added this to make sure. If you say `".txt"` would clean it we can remove it.

Reply:
Yeah, `fn_clean_exts` removes the string and everything after it, wherever it appears in the sample name. `fn_clean_trim` only removes the string if it's at the end. We have `.txt` in the former so it should strip the `_summary` as well.

I removed this before merge 👍🏻